root-packages/nexttrace/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/nxtrace/Ntrace-V1
 TERMUX_PKG_DESCRIPTION="An open source visual routing tool that pursues light weight"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.4.3-beta.1"
-TERMUX_PKG_SRCURL=https://github.com/nxtrace/Ntrace-V1/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=a35ca994839878b2502600b84134a3b90ff85b25b1609c11fcbfe692828535b2
+TERMUX_PKG_VERSION="1.5.0-beta.1"
```

Comment:
As mentioned in https://github.com/termux/termux-packages/pull/27157#discussion_r2507025062 this needs to have a `~` instead of a `-` to be considered prior to 1.5.0.
```suggestion
TERMUX_PKG_VERSION="1.5.0~beta.1"
```